### PR TITLE
ISSUE #3472 - Remove from favourite tooltip label is wrong

### LINKED ISSUE #3472: Remove from favourite tooltip label is wrong
### Description
The star icon tooltip for favourites fed/container states "add to favourite" instead of "remove from favourite"

- This is happening in: staging
- This requires a hotfix to production: no 

### Steps to replicate
- Go to containers page;
- Hover the star icon of a favourite container.

### Current Behaviour
- Tooltip label states "Add to favourite"

### Expected Behaviour
- Tooltip label should state "Remove from favourite"

### Pictures/ Gifs of the error
image

### PR DESCRIPTION
**Pull request by The-Daniel**
This fixes #3472 

#### Description
- Moved the tooltip into the favourites checkbox component so that it is shared wherever it is used (containers list, federations list)

#### Test cases
Hover over tooltip on both containers list and federations list

### CODE REVIEW

File: frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx [hunk 1]
```diff
@@ -144,27 +143,19 @@ export const ContainerListItem = ({
...
-									container._id,
-									!!event.currentTarget.checked,
-								);
-							}}
-						/>
-					</Tooltip>
+					<FavouriteCheckbox
+						checked={container.isFavourite}
+						selected={isSelected}
+						onClick={(event) => {
```
Comment (sanmont3drepo) [inline frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx:149]: I guess we could also move the onclick stoppropagation bit inside favouritecheckbox?

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]

File: frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx (hunk 1 above)
Comment (The-Daniel) [reply frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx:149]: @sanmont3drepo Good idea. The onChange and checked could even be moved into the control if you supply the component with a federationOrContainer object ..?

Comment (sanmont3drepo) [reply frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx:149]: I think  this is good enough.

Comment (The-Daniel) [reply frontend/src/v5/ui/routes/dashboard/projects/containers/containersList/containerListItem/containerListItem.component.tsx]: Done

**Review by sanmont3drepo — APPROVED** [review]

**Pull request merged**